tests-spark310+/src/test/scala/com/nvidia/spark/rapids/Spark310ParquetWriterSuite.scala

```diff
@@ -167,8 +169,10 @@ class Spark310ParquetWriterSuite extends SparkQueryCompareTestSuite {
       }
     }
     val ser = new ParquetCachedBatchSerializer
+
     val producer = new ser.CachedBatchIteratorProducer[ColumnarBatch](cbIter, schema,
-      new Configuration(true), new SQLConf)
+      SparkContext.getOrCreate().broadcast(new SerializableConfiguration(new Configuration(true))),
```

Comment:
curios why `withGpu/CpuSparkSession { sparkSession => sparkSession.sparkContext ...` is not used here

Reply:
No reason other than I forgot to use them as I haven't written a scala test in our plugin in a long time.